backup-restore/hotfixes/2.8.1/applypatch-281.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+# Run this script to update default datamover pod resource limit on spoke cluster.
+
+LOG=/tmp/applypatch-281_$$_log.txt
+exec &> >(tee -a $LOG)
+echo "Logging output in $LOG"
+
+# Check for Backup & Restore namespace
+BR_NS=$(oc get dataprotectionagent -A --no-headers -o custom-columns=NS:metadata.namespace)
+ISF_NS=$(oc get spectrumfusion -A -o custom-columns=NS:metadata.namespace --no-headers)
+
+if [ -z "$ISF_NS" ]; then
+    echo "ERROR: No Successful Fusion installation found. Exiting."
+    exit 1
+fi
+
+if [ -n "$BR_NS" ]; then
+    echo "This is Spoke."
```

Comment:
Is the intention to apply this on spoke only? What makes it not applicable on hub? Hub also includes agent. guardian-configmap and would have same setting for data mover Hence incorrect to log that "This is a spoke".

Reply:
right, as stated in #40578 i, on hub the resource limitation are already well updated.  Only on spoke, the resource limitation is wrong.